### ISSUE TITLE
createRenderTarget crashes

### ISSUE THREAD

**Issue by RblSb** [Kode/Krom#137]
`Segmentation fault: 11`
```haxe
var img:Image;
...
this.img = Image.createRenderTarget(20, 20);
var g = this.img.g2;
g.begin(true, 0xff000000);
g.fillCircle(20, 20, 20);
g.end();
```
Without `fillCircle`: `-[_MTLCommandEncoder dealloc]:75: failed assertion `Command encoder released without endEncoding'`

Without this block works, but crashes with krom `--watch` arg on Empty project.
macos 10.14.6

**Comment by RobDangerous** [comment]
Did you compile an up to date Krom version?

**Comment by RblSb** [comment]
Tested artifact from my PR, very latest.

**Comment by RobDangerous** [comment]
Try to build one with the latest Kinc revision.

**Comment by RblSb** [comment]
macOS CI has same error as here https://github.com/Kode/Krom/issues/129, but Linux compiles =)

**Comment by RobDangerous** [comment]
Uhm, what? But that error is in Chakra and that's still the same.

**Comment by RblSb** [comment]
Maybe Kinc wants clang v8 that is broken with old version of Chakra?

**Comment by RobDangerous** [comment]
No, it does not set a clang version. Will give it a try myself next time I boot macOS.
